test/src/examples_complete_test.go

```diff
@@ -93,3 +93,33 @@ func TestExamplesCompleteDisabled(t *testing.T) {
 	// Should complete successfully without creating or changing any resources
 	assert.Contains(t, results, "Resources: 0 added, 0 changed, 0 destroyed.")
 }
+
+func TestExamplesCompleteDefaults(t *testing.T) {
+    t.Parallel()
+    randID := strings.ToLower(random.UniqueId())
+	attributes := []string{randID}
+
+	rootFolder := "../../"
+	terraformFolderRelativeToRoot := "examples/complete"
+	varFiles := []string{"fixtures.defaults.tfvars"}
+
+    tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, rootFolder, terraformFolderRelativeToRoot)
+
+    terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: tempTestFolder,
+		Upgrade:      true,
+		// Variables to pass to our Terraform code using -var-file options
+		VarFiles: varFiles,
+		Vars: map[string]interface{}{
+			"attributes": attributes,
+			"enabled":    "false",
```

Comment:
Looks like you copied from the `TestExamplesCompleteDisabled` test instead of `TestExamplesComplete`

Reply:
Oh you're right. 1m

Reply:
@cdobbyn maybe we should set those inputs as `true` and put it in the existing fixtures so a new test doesnt have to be added ?

Reply:
@nitrocode I was thinking of coverage over both cases, but if you're comfortable with that I can do that.